specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 1036:
```
  cvr: TYPE = set[vote]
```
add in `ballot_manifest_info`
We want that info for `verified?`, `compare_reported_markings_to_cvr`, `cvr_table`.
The only related change would seem to be to drop `ballot_manifest_info` from  `ballots_under_audit_table`, since it will already be in `cvr`